devicemodel/samples/launch_ubuntu.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright (C) 2020 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+
+function launch_win()
```

Comment:
I know it does not change the functionality at all, but I would prefer to see a more consistent name used for this function, either aligned with the script, or a generic one (e.g. `launch_vm`)